example/bbs-dock.js

```diff
@@ -63,8 +65,9 @@ const residentCardSchema = {
 };
 
 const embeddedSchema = {
-  id: `data:application/json;charset=utf-8,${encodeURIComponent(JSON.stringify(residentCardSchema))}`,
+  id,
   type: 'JsonSchemaValidator2018',
+  details: stringify({ jsonSchema: residentCardSchema }),
```

Comment:
if the ID references the schema now (i.e can be resolved) - why do we still need to stringify it here?

Reply:
This example is showing an embedded schema. I have set the `id` to the embedded schema's id so that schema could be identified by PEX filter. [Here](https://github.com/docknetwork/sdk/pull/435/files#diff-255c77598b22bdb3552345ef44b6f544957abf45ea0bc32027224391f83594f4R91) is the test using external schema.